### ISSUE TITLE
Confusing CMake warning: -- igl_copy_dll() was called with a non-target: test_igl_copyleft_cgal

### ISSUE THREAD

**Issue by alecjacobson** [libigl/libigl#2352]
I noticed:

```
-- igl_copy_dll() was called with a non-target: test_igl_copyleft_cgal
```

in the Windows CI build. I'm a bit confused because this line https://github.com/libigl/libigl/blob/fe65ecb907730f96d99a92ebd9c5e852633cc990/cmake/igl/modules/copyleft/cgal.cmake#L32-L33

seems to check that `test_igl_copyleft_cgal` is a target but then in the function it's determined that it's not a target:

https://github.com/libigl/libigl/blob/fe65ecb907730f96d99a92ebd9c5e852633cc990/cmake/igl/igl_copy_dll.cmake#L53-L55

**Comment by jdumas** [comment]
The message (which really should be a warning) is here to warn users against potential misuse of the function. But in the case of the `test_igl_copyleft_cgal` we simply want to copy the dll when the unit test target exist (i.e. when `LIBIGL_BUILD_TUTORIALS` is `ON`). If we called the function without checking if `test_igl_copyleft_cgal` was a target we'd have a warning message when `LIBIGL_BUILD_TUTORIALS` is `OFF`.

**Comment by jdumas** [comment]
Oh ok I think the line should be this instead:
```
if(NOT TARGET ${target})
```

**Comment by alecjacobson** [comment]
Fixed in #2351
